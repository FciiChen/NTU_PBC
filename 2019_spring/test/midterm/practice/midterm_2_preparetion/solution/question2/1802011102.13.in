1B4, QVBayNoJ,n25ahZhaC1,meYjLq7e,akOnGWJqdb,IRiW,2Hi9KVJ,RdFPEiH
ZVgTm, 4Ij,lucOwR0ls,utQ8,PHD,qwaUAI0lme,nOzZ0ff1u,QUm21Fb1B5
BREAK
ZVgTm ,  bHRI1Rax
BREAK
