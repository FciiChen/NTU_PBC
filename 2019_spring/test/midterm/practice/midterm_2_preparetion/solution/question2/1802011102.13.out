1B4        QVBayNoJ   n25ahZhaC1 meYjLq7e   akOnGWJqdb IRiW       2Hi9KVJ    RdFPEiH    NULL      
ZVgTm      4Ij        lucOwR0ls  utQ8       PHD        qwaUAI0lme nOzZ0ff1u  QUm21Fb1B5 bHRI1Rax  
